src/main/java/github/grace000/erversay/RouteHandlers/Put.java

```diff
@@ -0,0 +1,89 @@
+package github.grace000.erversay.RouteHandlers;
+
+import github.grace000.erversay.Request.Request;
+import github.grace000.erversay.Response.Response;
+import github.grace000.erversay.Response.ResponseBuilder;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+import static github.grace000.erversay.Constants.Headers.CONTENT_LENGTH;
+import static github.grace000.erversay.Constants.Headers.PUT_HEADER;
+import static github.grace000.erversay.Constants.StatusCodes.NOT_ALLOWED_STATUS;
+
+public class Put implements RouteHandler{
+    private ResponseBuilder responseBuilder = new ResponseBuilder();
+    private String absolutePath = "public/sample.txt";
+    private List<String> resource = new ArrayList<>();
+    private Response response;
+
+    public enum AcceptedMethods {
+       GET, PUT
+    }
+
+    public boolean isMethodAllowed(String method) {
+        for (AcceptedMethods accepted: AcceptedMethods.values()) {
+            if (accepted.name().equals(method)){
+                return true;
+            }
```

Comment:
`accepted.name().equals(method)` returns a boolean. You don't need to do a hard return of `true` here.

Reply:
I tried refactoring this before by deleting the hard true, but I received compilation errors.

Reply:
Did you do a hard return on the `accepted.name().equals(method)`?